api/topicservice.yml

```diff
@@ -214,6 +214,24 @@ paths:
 
 components:
   schemas:
+    TitlesDTO:
+      type: object
+      properties:
+        default:
```

Comment:
hm, shouldn't it just use topic name and description instead of moving these attributes here (short, long, default)?